### PR TITLE
[KOGITO-2563] Add Red Hat headers to all sources in Kogito runtimes

### PR DESCRIPTION
**Pull request by bxf12315**
Many thanks for submitting your Pull Request :heart:! 

Please make sure that your PR meets the following requirements:

- [ ] You have read the [contributors guide](CONTRIBUTING.md)
- [ ] Pull Request title is properly formatted: `KOGITO-XYZ Subject`
- [ ] Pull Request title contains the target branch if not targeting master: `[0.9.x] KOGITO-XYZ Subject`
- [ ] Pull Request contains link to the JIRA issue
- [ ] Pull Request contains link to any dependent or related Pull Request
- [ ] Pull Request contains description of the issue
- [ ] Pull Request does not include fixes for issues other than the main ticket

### CODE REVIEW

**Comment by bxf12315** [comment]
@cristianonicolai please review it

File: src/resources/META-INF/services/com.mycila.maven.plugin.license.PropertiesProvider [hunk 1]
```diff
@@ -0,0 +1,3 @@
+com.mycila.maven.plugin.license.git.CopyrightRangeProvider
```
Comment (cristianonicolai) [inline src/resources/META-INF/services/com.mycila.maven.plugin.license.PropertiesProvider:1]: do we still need these entries, as it nows use `copyrightCreationYear` ?

Comment (bxf12315) [reply src/resources/META-INF/services/com.mycila.maven.plugin.license.PropertiesProvider:1]: it's default one, I can remov it.

**Comment by ricardozanini** [comment]
Can we close or resolve this one?

**Comment by cristianonicolai** [comment]
@bxf12315 planning on update this PR? otherwise we should close it.

**Comment by r00ta** [comment]
Hi @bxf12315   , can we close this PR or is it something still to be done? Thanks

**Comment by bxf12315** [comment]
close it.

**Pull request closed without merging**